modules/smoothxg/main.nf

```diff
@@ -0,0 +1,35 @@
+def VERSION = '0.6.1' // Version information not provided by tool on CLI
```

Comment:
Please update to v0.6.5. https://anaconda.org/bioconda/smoothxg. This should also come with a parameter which prints out the version of `smoothxg`.

Reply:
Cool, will do!

Reply:
Updated, will still need to add the bit to ask smoothxg to provide its own version

Reply:
Ironically, it's written to stdout:

```
smoothxg --version 2>&1 | cut -f 1 -d '-' | cut -f 2 -d 'v'
```

Will do the trick.

Reply:
https://github.com/pangenome/smoothxg/issues/171